src/nodes/NodeConnectionManager.ts

```diff
@@ -1278,9 +1287,15 @@ class NodeConnectionManager {
         requestSignature: requestSignature,
         relaySignature: relaySignature.toString('base64url'),
       });
-    }).finally(() => {
-      this.activeSignalFinalPs.delete(connProm);
-    });
+    })
+      // Ignore results and failures, then are expected to happen and are allowed
+      .then(
+        () => {},
+        () => {},
+      )
```

Comment:
I should really only ignore connection specific failures here. I'm going to fix that quickly.

Reply:
Addressed, I'm having it check if the error is connection related and throwing if not.